apps/community/src/i18n/en.ts

```diff
@@ -130,8 +130,8 @@ export default {
   },
   hooks: {
     register_dialog: {
-      title: '😔 Sorry you are not qualified!',
-      description: `If you would like to apply for access to "mail<sup>3</sup> Community". \n Please send us an email via the "Mail me" button below, telling us about your project, <span>your needs and your contact information</span>, and we will contact you after evaluation.`,
+      title: 'Apply for your Community',
+      description: `Please click the "mail me" button below to submit your application. It may take some time for your application to receive a response.`,
```

Comment:
```suggestion
      description: `Please click the "mail<sup>3</sup> me" button below to submit your application. It may take some time for your application to receive a response.`,
```